rb/lib/selenium/webdriver/common/service.rb

```diff
@@ -36,17 +36,31 @@ class Service
       SOCKET_LOCK_TIMEOUT = 45
       STOP_TIMEOUT        = 20
 
+      @executable = nil
+      @missing = ''
+
+      class << self
+        attr_accessor :executable, :missing_text
```

Comment:
Do we mean to let anyone outside the subclasses to set up the `missing_text`?
If the answer is no, then I'd suggest to have two different ones:
```ruby
class << self
  attr_reader :missing_text
  protected
  attr_writer :missing_text
end
```